src/traces/image/defaults.js

```diff
@@ -47,6 +47,7 @@ module.exports = function supplyDefaults(traceIn, traceOut) {
         traceOut.zmax = cm.zmaxDflt;
     }
 
+    coerce('interpolate', 'nearest');
     coerce('text');
```

Comment:
We should keep the constant 'dflt' in the attribute as it does not depend on anything.

Reply:
👍